news/2024/2024-09-04-osumania-4k-world-cup-2024-round-of-16.md

```diff
@@ -0,0 +1,121 @@
+- [Camellia - Antineutrino Witchcraft](https://osu.ppy.sh/beatmapsets/2238682#mania/4757059) (TB) @ [1:07:27](https://www.twitch.tv/videos/2238887961?t=01h07m27s) - This was it, whoever wins this wins the entire match. The map began, many combos broke left and right with the scores being so close to each other. However, someone's combo never broke… and that person was ::{ flag=SG }::[TheOPmeme](https://osu.ppy.sh/users/15763622), with them carrying their team to victory getting an FC on the tiebreaker with an incredible 988k score. With that, Singapore had done the impossible and had defeated the United Kingdom.
+
+This match was something else other than a regular match, this was a legendary showdown. Both teams played amazing with Singapore managing to cause an upset, and sending the United Kingdom to the losers bracket. In the quarterfinals, Singapore will have to face their toughest match-up yet going up against the #3 seed, Thailand. As for the United Kingdom, they will be going up against the #17 seed, Australia, in an attempt to keep their position in the tournament.
+
+### ::{ flag=CN }:: China vs. ::{ flag=MY }:: Malaysia
+
+[**You can view the match information here!**](https://osu.ppy.sh/community/matches/115310225)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2238945469&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+China and Malaysia are quite familiar with each other, they've provided many classic matches before, and by coincidence they met again this time. China has gathered a strong team in the mid-game and try to keep good consistency on every map, while Malaysia is well known as the early game being their skill set, especially for LN maps. As expected, this match was a show.
+
+- [Reizoko Cj - /a/](https://osu.ppy.sh/beatmapsets/2238283#mania/4755965) (RC6) @ [35:18](https://www.twitch.tv/videos/2238945469?t=00h35m18s) - This map is quite technical and risky, with roll patterns and long jack combinations making it hard for even the top players to keep good accuracy. China picked this map since they were favoured in the rice category, however that backfired when they lost some accuracy in the roll parts, meanwhile the Malaysians had 3 players with no 200s, which netted them their first break point.
```

Comment:
```suggestion
- [Reizoko Cj - /a/](https://osu.ppy.sh/beatmapsets/2238283#mania/4755965) (RC6) @ [35:18](https://www.twitch.tv/videos/2238945469?t=00h35m18s) - This map is quite technical and risky, with roll patterns and long jack combinations making it hard for even the top players to keep good accuracy. China picked this map since they were favoured in the rice category, however that backfired when they lost some accuracy in the roll parts. Meanwhile, the Malaysians had 3 players with no 200s, which netted them their first break point.
```